cargo-nextest/src/dispatch.rs

```diff
@@ -1579,4 +1580,21 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_negative_test_threads() {
+        let neg_j = "cargo nextest run --jobs -3";
+        let _app =
+            CargoNextestApp::try_parse_from(shell_words::split(neg_j).expect("valid command line"))
+                .unwrap();
+
+        let pos_j = "cargo nextest run --jobs 3";
+        let _app =
+            CargoNextestApp::try_parse_from(shell_words::split(pos_j).expect("valid command line"))
+                .unwrap();
+
+        let zero_j = "cargo nextest run --jobs 0";
+
+        assert!(CargoNextestApp::try_parse_from(shell_words::split(zero_j).unwrap()).is_err());
+    }
```

Comment:
These test vectors can just go in `test_argument_parsing` -- no need for another function.